extensions/wikia/GlobalNavigation/styles/GlobalNavigationInverse.scss

```diff
@@ -56,8 +57,15 @@ $global-nav-inverse-background-hover: #656E78;
 				border-radius: 50%;
 			}
 
-			.sign-in-label {
+			.auth-label,
+			.auth-link {
 				color: $global-nav-inverse-text-color;
+				&.register,
+				&.sign-in {
```

Comment:
Seems like there's no easy way of simplifying selector nesting without getting our properties overriden by the stronger ones.